### PR TITLE
Remove osdeps directory

### PR DESCRIPTION
**Pull request by Dhwaniartefact**
Removal of osdeps directory since it became unwanted after Archivematica 1.15

Connected to : https://github.com/archivematica/Issues/issues/1659

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -34,13 +34,15 @@ RUN set -ex \
...
 		locales-all \
+        media-types \
 		openssh-client \
 		p7zip-full \
 		rsync \
 		unar \
 		unzip \
 		xz-utils tk-dev \
 		zlib1g-dev \
+        rclone\
```
Comment (replaceafill) [inline Dockerfile]: This seems to be using spaces instead of tabs like the rest?

File: Dockerfile [hunk 2]
```diff
@@ -82,14 +84,10 @@ RUN set -ex \
...
 
-# Install Ubuntu release specific packages.
-COPY osdeps.py /src/osdeps.py
-COPY osdeps /src/osdeps
-
 USER root
 RUN set -ex \
 	&& apt-get update \
-	&& /src/osdeps.py Ubuntu-${UBUNTU_VERSION} 1 | grep -v -E "python3.9-dev" | xargs apt-get install -y --no-install-recommends \
+	&& Ubuntu-${UBUNTU_VERSION} 1 | grep -v -E "python3.9-dev" | xargs apt-get install -y --no-install-recommends \
```
Comment (replaceafill) [inline Dockerfile]: Is this line still necessary?

File: Dockerfile (hunk 1 above)
Comment (Dhwaniartefact) [reply Dockerfile]: Done. Oops my editor :(

File: Dockerfile (hunk 2 above)
Comment (Dhwaniartefact) [reply Dockerfile]: Thanks for the suggestion, removed now.

File: Dockerfile [hunk 3]
```diff
@@ -120,4 +128,4 @@ ENV DJANGO_SETTINGS_MODULE storage_service.settings.production
 
 EXPOSE 8000
 
-ENTRYPOINT pyenv exec python${PYTHON_VERSION} -m gunicorn --config=/etc/archivematica/storage-service.gunicorn-config.py storage_service.wsgi:application
+ENTRYPOINT pyenv exec python${PYTHON_VERSION} -m gunicorn --config=/etc/archivematica/storage-service.gunicorn-config.py storage_service.wsgi:application
```
Comment (replaceafill) [inline Dockerfile:130]: Is this change necessary?

File: Dockerfile [hunk 4]
```diff
@@ -1,9 +1,23 @@
 ARG TARGET=archivematica-storage-service
 ARG UBUNTU_VERSION=22.04
 
-FROM ubuntu:${UBUNTU_VERSION} AS base
+# Ubuntu osdeps install
```
Comment (replaceafill) [inline Dockerfile]: Is this comment necessary?

**Review by replaceafill — APPROVED** [review]
Looks good, but left a couple of questions.

File: Dockerfile (hunk 4 above)
Comment (Dhwaniartefact) [reply Dockerfile]: let me change it. Thank you !

File: Dockerfile (hunk 3 above)
Comment (Dhwaniartefact) [reply Dockerfile:130]: Not really, it seems editor issue.

Comment (replaceafill) [reply Dockerfile:130]: Was this change added to the `qa/0.x` branch?

Comment (Dhwaniartefact) [reply Dockerfile:130]: yes

Comment (replaceafill) [reply Dockerfile:130]: Was it intended? I understood the change was unnecessary.

Comment (Dhwaniartefact) [reply Dockerfile:130]: Not really.

Comment (replaceafill) [reply Dockerfile:130]: Cool. No problem, but I suggest you before requesting reviews or merging PRs to always check the `Files changed` tab of your pull requests. It should only contain your *intended* changes and no more.

Comment (replaceafill) [reply Dockerfile:130]: I think for this specific case we could use the [end-of-file-fixer](https://github.com/pre-commit/pre-commit-hooks?tab=readme-ov-file#end-of-file-fixer) from `pre-commit`.

Comment (Dhwaniartefact) [reply Dockerfile:130]: sure and thanks for the suggestions and notes.
